daemon/exec.go

```diff
@@ -88,6 +89,15 @@ func (d *Daemon) ContainerExecCreate(config *runconfig.ExecConfig) (string, erro
 	cmd := stringutils.NewStrSlice(config.Cmd...)
 	entrypoint, args := d.getEntrypointAndArgs(stringutils.NewStrSlice(), cmd)
 
+	keys := []byte{}
+	if config.DetachKeys != "" {
+		keys, err = term.ToBytes(config.DetachKeys)
+		if err != nil {
+			logrus.Warnf("%s", err.Error())
```

Comment:
I put two `Warnf`, not sure if it's needed. Maybe I should merge them. `err` will be a message like "Unknown character: …".